velox/exec/tests/SpillerTest.cpp

```diff
@@ -217,28 +357,133 @@ TEST_P(SpillerTest, error) {
   testSpill(100, true);
 }
 
-#ifndef NDEBUG
-/// This test verifies if the spilling partition is incrementally added.
-TEST_P(SpillerTest, incrementalSpillRunCheck) {
-  if (GetParam() != 8) {
-    // Test with spill execution pool size of 8 is sufficient and skip the other
-    // test settings.
+TEST_P(SpillerTest, spillWithEmptyPartitions) {
+  // kOrderBy type which has only one partition, is not relevant for this test.
+  if (type_ == Spiller::Type::kOrderBy) {
     GTEST_SKIP();
```

Comment:
nit: perhaps, add a message to explain why the test is skipped

Reply:
It is on top of if condition. I moved it right above GTEST_SKIP()